docs/GettingStarted.md

```diff
@@ -36,12 +36,12 @@ block { display: none; }
   display: block;
 }</style>
 <span>Platform:</span>
-<a href="javascript:void(0);" class="button-ios" onclick="display('platform', 'ios')">iOS</a>
-<a href="javascript:void(0);" class="button-android" onclick="display('platform', 'android')">Android</a>
+<a class="button-ios" onclick="display('platform', 'ios')">iOS</a>
+<a class="button-android" onclick="display('platform', 'android')">Android</a>
 <span>OS:</span>
-<a href="javascript:void(0);" class="button-mac" onclick="display('os', 'mac')">Mac</a>
-<a href="javascript:void(0);" class="button-linux" onclick="display('os', 'linux')">Linux</a>
-<a href="javascript:void(0);" class="button-windows" onclick="display('os', 'windows')">Windows</a>
+<a class="button-mac" onclick="display('os', 'mac')">Mac</a>
+<a class="button-linux" onclick="display('os', 'linux')">Linux</a>
+<a class="button-windows" onclick="display('os', 'windows')">Windows</a>
```

Comment:
I only changed this line and above. Not sure why Github is including other changes in this PR...